### PR TITLE
fix(webauthn): delete doesn't require challenge and use enroll wrapper

### PR DESCRIPTION
**Pull request by Alkorin**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Fix security keys

### CODE REVIEW

**Review by lizardK — DISMISSED** [review]

**Review by lizardK — APPROVED** [review]

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/account/user/security/u2f/user-security-u2f.constants.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const WEBAUTHN_URL = {
+  EU: 'https://auth.eu.ovhcloud.com/webauthn/enroll',
+  CA: 'https://auth.ca.ovhcloud.com/webauthn/enroll',
+  US: 'https://auth.us.ovhcloud.com/webauthn/enroll',
+};
+
+export default WEBAUTHN_URL;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/user/security/u2f/user-security-u2f.constants.js:7]:
```suggestion
export default {
    WEBAUTHN_URL,
 }
```

File: packages/manager/apps/dedicated/client/app/account/user/security/u2f/delete/user-security-u2f-delete.controller.js [hunk 2]
```diff
@@ -77,34 +73,7 @@ export default [
...
-          }
-          $scope.u2f.isLoading = false;
-          $scope.u2f.hasError = true;
-          Alerter.error($translate.instant(key), 'doubleAuthAlertU2fDelete');
-        })
-        .finally(() => {
-          $scope.u2f.isLoading = false;
-        });
-    };
+    $scope.init = () => {};
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/user/security/u2f/delete/user-security-u2f-delete.controller.js:76]: **nitpick:** Empty function can be removed. 
Also don't forget to remove it from Line number 10 on `packages/manager/apps/dedicated/client/app/account/user/security/u2f/delete/user-security-u2f-delete.html`
```suggestion
   
```

Comment (Alkorin) [reply packages/manager/apps/dedicated/client/app/account/user/security/u2f/delete/user-security-u2f-delete.controller.js]: Done

File: packages/manager/apps/dedicated/client/app/account/user/security/u2f/user-security-u2f.constants.js (hunk 1 above)
Comment (Alkorin) [reply packages/manager/apps/dedicated/client/app/account/user/security/u2f/user-security-u2f.constants.js]: Done

**Review by ghyenne — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Comment by gegtor** [comment]
Screenshot 2023-11-07 at 15 05 00
YubiKeys Also

**Comment by gegtor** [comment]
> @gegtor weird, actually both 001 and 002 on my screenshot are Yubikey 5C NFC, You are allowed to rename your key on the dashboard / security ! Try to do that :)

It doesn't show up in dashboard but does while logging in

**Pull request merged**